jekyll/_includes/snippets/ja/machine-resource-table.md

```diff
@@ -1,7 +1,7 @@
-| クラス     | vCPU | RAM    |
-| ------- | ---- | ------ |
-| medium  | 2    | 7.5 GB |
-| large   | 4    | 15 GB  |
-| xlarge  | 8    | 32 GB  |
-| 2xlarge | 16   | 64 GB  |
+| クラス     | vCPU | RAM   |
+| ------- | ---- | ----- |
+| medium  | 2    | 100 GB |
+| large   | 4    | 100 GB |
+| xlarge  | 8    | 100 GB |
+| 2xlarge | 16   | 100 GB |
```

Comment:
I think the disk size were added .

```
| クラス     | vCPU | RAM    | ディスクサイズ   |
| ------- | ---- | ------ | ----- | 
| medium  | 2    | 7.5 GB | 100 GB |
| large   | 4    | 15 GB  | 100 GB |
| xlarge  | 8    | 32 GB  | 100 GB |
| 2xlarge | 16   | 64 GB  | 100 GB |

```

https://circleci.com/docs/2.0/executor-types/#using-machine
<img width="889" alt="image" src="https://user-images.githubusercontent.com/6015450/168260003-f9a998e5-8485-4636-9e78-101b28df3379.png">

Reply:
Thank you! I'll create a pull-request to fix En version as well!

Reply:
Ah, this has already been fixed in En file, I just have to fix Ja then.